kikar_hamedina/kikar_hamedina/settings.py

```diff
@@ -260,3 +260,21 @@
     from .local_settings import *
 except ImportError:
     pass
+
+if "TRAVIS" in os.environ:
+    DATABASES = {
+        'default': {
+            'ENGINE':   'django.db.backends.postgresql_psycopg2',
+            'NAME':     'travisci',
+            'USER':     'postgres',
+            'PASSWORD': '',
+            'HOST':     'localhost',
+            'PORT':     '',
+        }
+    }
+
+    DEBUG = True
+
+    FACEBOOK_APP_ID = '711830842197328'
+    FACEBOOK_SECRET_KEY = '772cec8e6b20b02104e4a0c6bb7fad9e'
+    FACEBOOK_API_VERSION = 'v2.5'
```

Comment:
My test application credentials. We can have a specific application just for the continuous integration